### PR TITLE
[historical-view] Uses new backend endpoint and data format

### PR DESCRIPTION
**Pull request by madsnedergaard**
### Description

_part of #2393_

Switching to the new endpoint and new format in `/state`. For now it's only supporting 'hourly' directly, so we're filling the grid data from `/state` into `histories` in the Redux state.

This is a temporary solution to get a first prototype out and some initial scaffolding, as we're going to restructure the state completely to support multiple "aggregates/windows" (hourly, monthly, yearly) of data.

### CODE REVIEW

File: web/src/reducers/dataReducerForHistoryFeature.js [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        zone.renewableRatio = undefined;
+        zone.renewableRatioProduction = undefined;
+        zone.exchange = {};
+        zone.production = {};
+        zone.productionCo2Intensities = {};
+        zone.productionCo2IntensitySources = {};
+        zone.dischargeCo2Intensities = {};
+        zone.dischargeCo2IntensitySources = {};
+        zone.storage = {};
+        zone.source = undefined;
```
Comment (Kongkille) [inline web/src/reducers/dataReducerForHistoryFeature.js:104]: Here we could only reset what we're getting in state, eg zoneStateOverview

   {
            "countryCode": zone_key,
            "stateDatetime": format_utc_datetime(state_datetime),
            "estimationMethod": estimation_method,
            "co2intensity": data["carbon_intensity_avg"],
            "co2intensityProduction": data["carbon_intensity_production_avg"],
            "fossilFuelRatio": percentage_to_ratio(
                data, "power_origin_percent_fossil_avg"
            ),
            "renewableRatio": percentage_to_ratio(
                data, "power_origin_percent_renewable_avg"
            ),
            "fossilFuelRatioProduction": percentage_to_ratio(
                data, "power_production_percent_fossil_avg"
            ),
            "renewableRatioProduction": percentage_to_ratio(
                data, "power_production_percent_renewable_avg"
            ),
        }

File: mockserver/server.js [hunk 2]
```diff
@@ -17,6 +19,17 @@ app.get("/v4/history", (req, res, next) => {
...
+app.use(function (req, res, next) {
+  // Get rid of query parameters so we can serve static files
+  if (Object.entries(req.query).length !== 0) {
+      res.redirect(url.parse(req.url).pathname);
+  } else {
+    // Log all requests to static files
+    console.log(req.method, req.path)
+    next();
+  }
+});
```
Comment (Kongkille) [inline mockserver/server.js:31]: Nice!

**Comment by madsnedergaard** [comment]
Tested and works out of the box (except a tiny copy-paste mistake I did) with actual v5/state 😍 
@Kongkille

**Review by Kongkille — APPROVED** [review]
Sorry didn't realize i was blocking here

**Pull request merged**